wal/src/rocks_impl/config.rs

```diff
@@ -7,17 +7,31 @@ use serde::{Deserialize, Serialize};
 #[derive(Debug, Clone, Serialize, Deserialize)]
 #[serde(default)]
 pub struct Config {
+    pub max_subcompactions: u32,
     pub max_background_jobs: i32,
     pub enable_statistics: bool,
+    pub write_buffer_size: u64,
+    pub max_write_buffer_number: i32,
+    pub level_zero_file_num_compaction_trigger: i32,
+    pub level_zero_slowdown_writes_trigger: i32,
+    pub level_zero_stop_writes_trigger: i32,
```

Comment:
Can these fields be negative? If they can, maybe we should add some comments to explain the negative values meaning. Otherwise, how about using usize here?

Reply:
The datatypes is same as rocksdb itself. I add some comments.